kroma-chain-ops/genesis/layer_two_test.go

```diff
@@ -90,7 +90,7 @@ func TestBuildL2MainnetGenesis(t *testing.T) {
 	config.EnableGovernance = true
 	config.FundDevAccounts = false
 	gen := testBuildL2Genesis(t, config)
-	require.Equal(t, 526, len(gen.Alloc))
+	require.Equal(t, 528, len(gen.Alloc))
```

Comment:
Could you explain this?

Reply:
precompile count = 256
non-proxy predeploys = 1
proxy predeploys = 256
proxy implementations = 15
= 528

If GovernanceEnabled() is false, subtract 2 from the num of proxy implementations.
= 526